database/index.js

```diff
@@ -17,4 +17,61 @@ const pool = new Pool({
   database: 'fetsy',
 });
 
+const model = {
```

Comment:
i might separate this out into a model folder exactly as is. Just to emphasize separation of concerns (dbconnection, model)